### ISSUE TITLE
v2.4.7 bug - Calling meteorMethod fails one new doc when initialized with _id 

### ISSUE THREAD

**Issue by s7dhansh** [jagi/meteor-astronomy#630]
If I do something like,
```
let u = new Test({_id: Random.id()});
u.testMethod();
```

I get an error,
```
TypeError: Cannot read property 'set' of undefined
    at [object Object].astronomyExecute (packages/jagi:astronomy/lib/modules/methods/meteor_methods/astronomyExecute.js:18:5)
    at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1737:12)
    at packages/ddp-server/livedata_server.js:1666:15
    at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
    at packages/ddp-server/livedata_server.js:1664:30
    at [object Object]._.extend.applyAsync (packages/ddp-server/livedata_server.js:1663:12)
    at [object Object]._.extend.apply (packages/ddp-server/livedata_server.js:1611:26)
    at callStandardMeteorMethod (packages/jagi:astronomy/lib/modules/storage/utils/callStandardMeteorMethod.js:7:21)
    at Class.applyMethod (packages/jagi:astronomy/lib/modules/methods/class_prototype_methods/applyMethod.js:18:10)
    at Class.testMethod (packages/jagi:astronomy/lib/modules/methods/utils/wrapMethod.js:23:16)
```

I am doing this only on server, so not sure if it happens on client.

This works when I use `u= new Test()` instead of `u = new Test({_id: Random.id()})`.

**Comment by lukejagodzinski** [comment]
Hmmm, I've been recently improving how Meteor methods work in Astronomy and I might introduce a bug. I will take a look at it later today.

**Comment by lukejagodzinski** [comment]
Bug fixed in 2.4.8
